stylelint.config.js

```diff
@@ -1,6 +1,9 @@
 module.exports = {
   extends: '@jetbrains/stylelint-config',
+  plugins: ['@csstools/stylelint-no-at-nest-rule'],
   rules: {
+    '@csstools/stylelint-no-at-nest-rule': true,
```

Comment:
It's up for discussion :)

Reply:
Since it protects us from unwanted syntax usage, I don't see reason against that rule : )